conf/LocalSettings.php

```diff
@@ -119,71 +121,42 @@
 $wgDiff3 = "/usr/bin/diff3";
 
 ## Default skin: you can change the default skin. Use the internal symbolic
-## names, ie 'vector', 'monobook':
+## names, e.g. 'vector' or 'monobook':
 $wgDefaultSkin = "vector";
 
 # Enabled skins.
 # The following skins were automatically enabled:
+wfLoadSkin( 'MinervaNeue' );
 wfLoadSkin( 'MonoBook' );
 wfLoadSkin( 'Timeless' );
 wfLoadSkin( 'Vector' );
 
+
 # End of automatically generated settings.
 # Add more configuration options below.
 
-# Enabled extensions. Most of the extensions are enabled by adding
-# wfLoadExtensions('ExtensionName');
-# to LocalSettings.php. Check specific extension documentation for more details.
+# LDAP Settings
+# See https://www.mediawiki.org/wiki/Manual:Active_Directory_Integration
 
-# LDAPProvider
-# https://www.mediawiki.org/wiki/Extension:LDAPProvider
+wfLoadExtension( 'LDAPAuthorization' );
+wfLoadExtension( 'LDAPAuthentication2' );
+wfLoadExtension( 'LDAPGroups' );
 wfLoadExtension( 'LDAPProvider' );
-
-# PluggableAuth
-# https://www.mediawiki.org/wiki/Extension:PluggableAuth
+wfLoadExtension( 'LDAPUserInfo' );
 wfLoadExtension( 'PluggableAuth' );
 
+$LDAPProviderDomainConfigs = "$IP/ldapproviders.json";
+$wgPluggableAuth_ButtonLabel = "Log In";
+$LDAPProviderDefaultDomain = "yunohost.local";
+
 $wgPluggableAuth_EnableLocalLogin = true;
 $wgPluggableAuth_EnableLocalProperties = true;
 
-# LDAPAuthentication2
-# https://www.mediawiki.org/wiki/Extension:LDAPAuthentication2
-wfLoadExtension( 'LDAPAuthentication2' );
 
-# Note(decentral1se): leaving here in case we need them one day
-# LDAPGroups
-# https://www.mediawiki.org/wiki/Extension:LDAPGroups
-# wfLoadExtension( 'LDAPGroups' );
-
-# Note(decentral1se): leaving here in case we need them one day
-# LDAPUserInfo
-# https://www.mediawiki.org/wiki/Extension:LDAPUserInfo
-# wfLoadExtension( 'LDAPUserInfo' );
-
-# Note(decentral1se): leaving here in case we need them one day
-# LDAPAuthorization
-# https://www.mediawiki.org/wiki/Extension:LDAPAuthorization
-# wfLoadExtension( 'LDAPAuthorization' );
-
-# LDAP configuration
-$LDAPProviderDomainConfigProvider = function() {
-  $config = [
-    "LDAP" => [
-      "connection" => [
-        "server" => "localhost",
-        "port" => "389",
-        "basedn" => "dc=yunohost,dc=org",
-        "groupbasedn" => "ou=users,dc=yunohost,dc=org",
-        "userbasedn" => "ou=groups,dc=yunohost,dc=org",
-        "searchattribute" => "uid",
-        "usernameattribute" => "uid",
-        "realnameattribute" => "cn",
-        "emailattribute" => "mail"
-      ]
-    ]
-  ];
-
-  return new \MediaWiki\Extension\LDAPProvider\DomainConfigProvider\InlinePHPArray( $config );
-};
-$createaccount = __CREATE_ACCOUNT__;
-$wgGroupPermissions['*']['createaccount'] = $createaccount;
```

Comment:
That tweak is needed for the config panel to work tho

Reply:
Yeah I renamed the option, I should’ve updated the config panel too…

Reply:
I reverted this change, should be merged at once.